migrations/2023-08-23-182533_scaled_rank/up.sql

```diff
@@ -0,0 +1,28 @@
+CREATE OR REPLACE FUNCTION scaled_rank (score numeric, published timestamp without time zone, users_active_month numeric)
+    RETURNS integer
+    AS $$
+BEGIN
+    -- Add 2 to avoid divide by zero errors
+    -- Use 0.1 to lessen the initial sharp decline at a hot_rank ~ 300
```

Comment:
This `0.1` scale factor is the "I made it up on the spot" part of this PR, and I don't know if it makes any sense. 

I tried factors of 1, 0.1, 0.01, using a graphing calculator and some regular hot ranks, and compared them with various community sizes. Using smaller numbers lessens the initial sharp decline sensitivity.